src/routes/ExercisesScreens.tsx

```diff
@@ -17,6 +20,17 @@ const Root = styled.View`
   background-color: ${prop => prop.theme.colors.background};
   height: 100%;
 `
+const SmallMessage = styled(HeadingText)`
+  width: ${wp('60%')}px;
+  margin-bottom: ${props => props.theme.spacings.md};
+  text-align: center;
+  font-size: ${prop => prop.theme.fonts.defaultFontSize};
+  font-weight: ${prop => prop.theme.fonts.lightFontWeight};
+`
+const BoldText = styled(HeadingText)`
```

Comment:
For this please add a new element to `Content.tsx` which is called `ContentTextBold` and use it.